plugins/optimization-detective/types.ts

```diff
@@ -69,11 +79,13 @@ export type FinalizeArgs = {
 		properties: ExtendedElementData
 	) => void;
 	readonly isDebug: boolean;
+	readonly logger: Logger;
 };
 
 export type FinalizeCallback = ( args: FinalizeArgs ) => Promise< void >;
 
 export interface Extension {
+	readonly consoleLogPrefix?: string;
```

Comment:
Instead of an extension exporting a `consoleLogPrefix`, how about they instead export a `name`? Then we can use it to construct the prefix. If no `name` is exported, then the prefix can be `[Optimization Detective Extension]`.

```suggestion
	readonly name?: string;
```